test/sqllogictest/explain/raw_plan_as_text.slt

```diff
@@ -103,9 +103,8 @@ query T multiline
 EXPLAIN RAW PLAN AS TEXT FOR
 VIEW ov
 ----
-Project (#0, #1)
-  TopK order_by=[#1 asc nulls_last, #0 desc nulls_first] limit=5
-    Get materialize.public.t
+TopK order_by=[#1 asc nulls_last, #0 desc nulls_first] limit=5
```

Comment:
I'm removing this test in #21383 because `EXPLAIN VIEW` is now deprecated, so you might want to revert those changes in order to avoid conflicts.